app/helpers/schema_helper.py

```diff
@@ -145,13 +134,12 @@ def get_aliases(cls, survey_json):
         for block in cls.get_blocks(survey_json):
             for question in cls.get_questions_for_block(block):
                 for answer in question['answers']:
-                    if 'alias' in answer and answer['alias'] not in aliases:
+                    if 'alias' in answer:
+                        assert answer['alias'] not in aliases, 'Duplicate alias found: ' + answer['alias']
```

Comment:
in theory asserts are removed from production code (you have to turn on the Python optimise flags) so it's a developer aid in case anything ever goes wrong (some code changes break it for example or the schema validator stopped working)